shell/packages/sandstorm-db/db.js

```diff
@@ -590,7 +592,7 @@ isSignedUpOrDemo = function () {
   return false;
 }
 
-var calculateReferralBonus = function(accountId, plan) {
+var calculateReferralBonus = function(user, plan) {
```

Comment:
It appears to me that getUserQuota() has been calling this with the incorrect second parameter all along -- it passes in a plan object, but this function expects a string name. Why does this work?